_posts/2023-11-14-release.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Release 2023.46
+author: Priya
+date: 2023-11-14
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+#### Centralized Tracking for Tasks with Data Sync Off:
+Introducing a new global area, users can now conveniently access a centralized list displaying all tasks where data sync is turned off.
```

Comment:
The global pending area in the navigation bar now has a new tab. Users can now conveniently access a  centralized list displaying all tasks where data sync is turned off.